apps/meteor/app/api/server/helpers/getMatrixInstances.ts

```diff
@@ -0,0 +1,4 @@
+import { makeFunction } from '@rocket.chat/patch-coordinator';
+import type { BrokerNode } from 'moleculer';
+
+export const getMatrixInstances = makeFunction(async (): Promise<BrokerNode[]> => []);
```

Comment:
why are we calling these matrixInstances?  this naming doesn't make sense ?  Wouldn't this almost literally be getRocketChatInstances?  

I mean I know it was this way already.  But if refactoring and moving it around.  Would be nice if reading it semantically made more sense

Reply:
changed it.